net-misc/freerdp/freerdp-2.9.0.ebuild

```diff
@@ -67,6 +68,7 @@ RDEPEND="
 			x11-libs/libXtst
 			xinerama? ( x11-libs/libXinerama )
 		)
+		ffmpeg? ( media-video/ffmpeg[encode,x264] )
```

Comment:
Why are you adding this dependency?

Reply:
ffmpeg is used to encode audio, if this is not enabled, the freerdp server cannot stream audio.
Look at the code in libfreerdp/codec/dsp_ffmpeg.c , class freerdp_dsp_ffmpeg_encode for example

Reply:
See also
https://github.com/FreeRDP/FreeRDP/issues/8686
build error with ffmpeg master git: libfreerdp/codec/h264_ffmpeg.c:618:55: error: ‘AV_CODEC_CAP_TRUNCATED’ undeclared